Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```diff
@@ -13,6 +13,10 @@
     - Hunger
     - Thirst
     - Idle
+  - type: Hunger
+  - type: Thirst
```

Comment:
I don't think mobs can even eat without hands, we need a component so they can consume floor food or the likes.